kuma/static/js/newsletter.js

```diff
@@ -5,23 +5,18 @@
 
     var newsletterForm = doc.getElementById('newsletterForm');
     var isHidden = false;
-    var isArticle = false;
+    var isArticle = window.location.pathname.indexOf('/docs/') !== -1;
 
     if(!newsletterForm) {
         return;
     } else {
         // check for local storage
-        if(mdn.features.localStorage) {
+        if(mdn.features.localStorage !== false) {
```

Comment:
On the beta site, mdn.features.localStorage is undefined, so there is a change here and below to look explicity for false. On the beta site we simply assume that localStorage is available.